### ISSUE TITLE
crash on 17995 when trying to play mario kart midnight v1.1

### ISSUE THREAD

**Issue by soccerboys2008** [flathub/org.DolphinEmu.dolphin-emu#143]
when trying to play mario kart midnight v1.1 riivolution patch (https://wiki.tockdom.com/wiki/Mario_Kart_Midnight)

i get this:
```
An error occurred.


IntCPU: Unknown instruction 00000000 at PC = 8025924c  last_PC = 8132ffc8  LR = 800060a8


  Condition: 0
  File: /run/build/dolphin-emu/Source/Core/Core/PowerPC/Interpreter/Interpreter.cpp
  Line: 347
  Function: unknown_instruction

Ignore and continue?
```
i booted up a windows vm and it worked fine there, so i cant tell if this is an issue with the linux version or the flatpak build

and i got the flatpak so i didn't have to build it

**Comment by soccerboys2008** [comment]
just compiled dolphin from source and it was an issue with dolphin itself, and since that was the case i opened an issue on there issue tracker. however i'm still gonna keep this open until a build of dolphin is released that fixes the issue and this flatpak is updated to reflect that change

dolhine issue reference: https://bugs.dolphin-emu.org/issues/13138

**Comment by soccerboys2008** [comment]
this is fixed in versions of dolphin >= 5.0-18748
